### DISCUSSION TITLE
Where should `Robyn` usually get installed on my machine?

### DISCUSSION THREAD

**Discussion by gedkitchener-CT** [facebookexperimental/Robyn#938] [General]
Hi, I am new to R, so apologies in advance for the noob question. Any help would be much appreciated.

I am using a Debian 11 OS.

I have run the following:

```R
remotes::install_version("Robyn", "3.7.2")
```

which installs 94 dependencies into `/usr/local/lib/R/site-library`. In that directory, I can see the dependencies: `curl`, `ggplot2`, `stringr`, etc.

But when I come to load the `Robyn` library in my script:

```R
library(Robyn)
```

I get the following error:

```
Error in library(Robyn): there is no package called ‘Robyn’
```

When I check the library path using `.libPaths()` I see that same `/usr/local/lib/R/site-library` directory, so it's looking in the right place for the dependencies. **_But I do notice that `Robyn` is not there alongside them._**

Should `Robyn` be in the same directory as the dependencies? Or is it installed elsewhere by default?

**Comment by gufengzhou** [comment]
Sorry for the late reply, because the discussion here is not as closely monitored. Can you check `installed.packages()[, 'LibPath']` and do you see the library Robyn there? If not, you should just reinstall it first.
